01-30-shield-blocking/actors/enemy/Enemy.gd

```diff
@@ -0,0 +1,26 @@
+extends KinematicBody2D
+class_name Enemy
+
+onready var animated_sprite : AnimatedSprite = $AnimatedSprite
+
+export var move_speed : = 75.0
```

Comment:
The move speed on both characters is low, making the controls not feel as tight as they could be